packages/react/src/components/Dropdown/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import ReactSelect from 'react-select';
+import Icon from '../Icon';
+import UiText from '../UiText';
+import { safeInvoke } from '../../utils';
+
+const customStyles = {
```

Comment:
Could you add those style via CSS instead of inline ones?

Reply:
I was considering it, but then it would require each attribute to be followed by `!important` to override original style. Do you think that would be better?

Reply:
Can it be achieved by increasing the CSS selector level?
`.f-Dropdown .foo.bar`?